src/routes/completion/+page.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+	import { useCompletion } from '@ai-sdk/svelte'
+
+	const { completion, input, handleSubmit, data } = useCompletion();
+</script>
+
+<svelte:head>
+	<title>Home</title>
+	<meta name="description" content="Svelte demo app" />
```

Comment:
be consistent and cleanup the code

Reply:
noted. will do a code cleanup and other changes we discussed in next iteration. Thankyou for this review.